MiniKeePass/Settings View/SettingsViewController.swift

```diff
@@ -226,6 +226,7 @@ class SettingsViewController: UITableViewController, PinViewControllerDelegate {
             let pinViewController = PinViewController()
             pinViewController.titleLabel.text = NSLocalizedString("Set PIN", comment: "")
             pinViewController.delegate = self
+            pinViewController.view.backgroundColor = UIColor.white
```

Comment:
Maybe add a short comment here describing why you're setting the background to white in the SettingsViewController.  It took me a little while to realize that it was because we wanted the background white for setting the PIN, but transparent so the UIWindow could blur it out in the lock screen.